src/profiles-tile.js

```diff
@@ -20,24 +19,24 @@ export default class ProfilesTile extends View {
 
 		this.subscriptions = new CompositeDisposable();
 
-		this.subscriptions.add(this.targetPlatform.on('click', () => {
+		// On click handler
+		this.targetPlatform.on('click', () => {
```

Comment:
No as using it previously was unnecessary. `CompositeDisposable` was disposing only `Disposable` instances and `.on()` and `_onCurrentTargetPlatformChanged()` don't return it anyway :)